reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -76,10 +78,21 @@ object SpdxDocumentModelMapper {
         ortResult.getPackages(omitExcluded = true).forEach { curatedPackage ->
             val pkg = curatedPackage.pkg
 
+            if (pkg.vcsProcessed.url.isNotBlank()) {
+                val purlExternalReference = SpdxExternalReference(
```

Comment:
Please extract line 82-86 into an extension function analog to e.g. `private fun VcsInfo.toSpdxDownloadLocation(): String { `below, for being more consistent in the abstraction level here in this caller.

Reply:
`pkg.vcsProcessed.url.isNotBlank()` was a copy-paste mistake fixed it now should have been `pkg.purl.isNotBlank()`